diff --git a/src/conv.rs b/src/conv.rs
index e00d35e..18e4a53 100644
--- a/src/conv.rs
+++ b/src/conv.rs
@@ -253,7 +253,7 @@ pub fn map_pipeline_layout_descriptor<'a>(
         .map(|range| wgt::PushConstantRange {
             stages: wgt::ShaderStages::from_bits(range.stages)
                 .expect("invalid shader stage for push constant range"),
-            range: range.start..range.end,
+            range: range.start..range.stop,
         })
         .collect()
     });
